packages/connect/src/core/AbstractMethod.ts

```diff
@@ -341,6 +341,9 @@ export abstract class AbstractMethod<Name extends CallMethodPayload['method'], P
     }
 
     checkDeviceCapability() {
+        if (!this.useDevice) {
+            return true;
+        }
```

Comment:
if cardanoComposeTransaction, or any method that has useDevice: false should also inherit from a coin specific abstract method, we need to handle cases like this. I am starting to dislike it